src/app/containers/NielsenAnalytics/Amp/index.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import { string } from 'prop-types';
+
+// styled-components removes non-standard attributes (such as AMP attributes) on
+// server rendering. spreading props like this allows us to add AMP attributes
+// to the element.
+const AccessDiv = props => <div {...props} />;
+
+// Nielsen should only run in AU,
+// Using amp-geo to hide/display based on the country
+// amp-analytics doesn't generate any call if display none
+const DisplayWrapper = styled(AccessDiv)`
+  .amp-geo-pending &,
+  .amp-geo-group-gbOrUnknown & {
+    display: none;
+    visibility: hidden;
+  }
+  .amp-iso-country-au & {
+    display: block;
+  }
```

Comment:
So this covers:
- Don't display when loading
- Don't display in GB or unknown location
- Display in australia

What stops the analytics element appearing in every other country outside GB? It perhaps needs a default `display: none; visibility: hidden` before the more specific selector for AU?

Reply:
Fixed